Makefile

```diff
@@ -0,0 +1,21 @@
+SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
+ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+
+null:
+	@echo Please specify a make target.
+
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
+	cd $(CHAIN) && go install -tags 'localhost_auth http_ok reset' ./cmd/cored
+
+## builds statically-linked c dependencies
+build-lib:
+	-mkdir $(SNAPPY)/build
+	cd $(SNAPPY)/build && cmake ..
+	cd $(SNAPPY)/build && $(MAKE)
+	cd $(ROCKSDB) && $(MAKE) static_lib
+
+## clean removes all data directories and c libraries
+clean:
+	rm -rf $(SNAPPY)/build
+	rm -f $(ROCKSDB)/librocksdb.a
```

Comment:
man, for some reason, I thought I explicitly added a newline earlier. I don't know where it went! 

(added.)